### PR TITLE
Load model choices from the admin form (Fix 52)

### PR DESCRIPTION
**Pull request by marksweb**
The import model's `model` field is currently loading settings for the available models for celery. This causes django to think migrations are required to encompass those choices, so you'll get asked to create a migration when you integrate the app into a project, and then everytime that setting changes.

This loads the choices in the form which avoid this.

### CODE REVIEW

File: import_export_celery/models/importjob.py [hunk 1]
```diff
@@ -71,9 +71,7 @@ class ImportJob(models.Model):
     model = models.CharField(
         verbose_name=_("Name of model to import to"),
         max_length=160,
-        choices=[
-            (x, x) for x in getattr(settings, "IMPORT_EXPORT_CELERY_MODELS", {}).keys()
-        ],
+        choices=[],
```
Comment (timthelion) [inline import_export_celery/models/importjob.py:74]: This line can be deleted, right?

**Comment by timthelion** [comment]
Seems like a reasonable change. Would you mind running black to make travis happy?

File: import_export_celery/models/importjob.py (hunk 1 above)
Comment (marksweb) [reply import_export_celery/models/importjob.py:74]: @timthelion Yes it can now that I've had to use a choice field on the form 👍

**Comment by marksweb** [comment]
@timthelion Yeah of course. I'm currently "down the rabbit hole" after running black.

Hope you don't mind, but I'm also adding pre-commit so that black will run before travis gets hit. In theory these things don't fail in future with that in place.

And after doing that I've added a few hooks that I find helpful for project maintenance, including a github action to run flake8.

So I've got a bit side-tracked thanks to travis, but I think it's good for the project. And I could update the setup config to drop old python and include new python because it's got python 3.3, 3.4 and 3.5 in there.

**Comment by marksweb** [comment]
@timthelion Ok I've finished and all checks pass.

Apologies for the amount of changes here, most are from automated tools that I've added via pre-commit.

**Comment by timthelion** [comment]
Seems fine. I personally don't use flake8 because I don't like manually removing newlines at the ends of files and other trivial pendanticisms, but for this project that'w won't bother me because I don't have to work on it very often...

**Comment by marksweb** [comment]
@timthelion ah ok. There was config for it so I assumed it was intended.

I could also add 2 github actions for pypi. One would test a merged PR with a build and upload to test.pypi.org and the other would actually release the app via github releases. They've made my package maintenance easier and just needs an access token for each.

**Comment by timthelion** [comment]
Ooo, autoreleasing would be a blessing. Back in the dark ages I set up GPG signing and verification with pypi, obviously the github releases will be unsigned... We'll see if pypi will accept the unsigned releases... I don't know what the policy is now, but at somepoint debiand did not accept unsigned upstream releases either, but in anycase we aren't packaged by debian, its just a thought that ocvured to me...

**Pull request merged**